### ISSUE TITLE
[1.12.2] rftools crashes server with blood filled blizz syringe in spawner

### ISSUE THREAD

**Issue by Morpheus1101** [McJty/RFTools#1981]
Issue description:  When placing a syringe filled of blizz blood the server on SMP or client on SSP crashes instantly

Steps to reproduce: 

Place down a spawner from rftools, open up its GUI an place a syringe of blood from a blizz inside and instant crash.

Versions:
* Minecraft:  1.12.2
* Forge:  *.5.2847
* McJtyLib:  3.5.4
* RFTools: 7.72

Relevant logs, if any:  Crash report @ https://pastebin.com/Lfq78Fcd

**Comment by Morpheus1101** [comment]
Looks like it may also be the same issue for #1967

**Comment by josephcsible** [comment]
Please post your rftools.cfg file. The problem is in there.

**Comment by Morpheus1101** [comment]
This is how it was at the crash time

[rftools.txt](https://github.com/McJtyMods/RFTools/files/3633169/rftools.txt)

**Comment by josephcsible** [comment]
Can you see if you can reproduce this with just RFTools, Thermal Foundation, and their dependencies?

**Comment by Morpheus1101** [comment]
With an updated config adding in the various spawning options for blizz, it works fine without crashing, it would seem due to no prev option for a recipe so to speak for the blizz that was causing the crash(at least in my case) 

TLDR; Many issues had cropped up, but re-generating config and re-adding the needed spawn options has solved everything.
